driver/src/clawfc/depscan/FortranIncludesResolver.g4

```diff
@@ -0,0 +1,56 @@
+/*
+ * This file is released under terms of BSD license
+ * See LICENSE file for more information
+ * @author Mikhail Zhigun
+*/
+/**
+ * ANTLR 4 Grammar file for detecting Fortran include statements.
+ */
+grammar FortranIncludesResolver;
+
+root : (include_line)* EOF;
+
+include_line : INCLUDE_STATEMENT_LINE;
+
+INCLUDE_STATEMENT_LINE : SEP? INCLUDE SEP INCLUDE_STRING SEP? EOL;
+OTHER_LINE : (~'\n')* EOL ->skip;
+
+fragment INCLUDE_STRING : (DQ (~'"' | QUOTED_DQ)* DQ)
+                | (SQ (~'\'' | QUOTED_SQ)* SQ);
+
+fragment SEP : WS+;
+fragment WS : [ \t];
+fragment EOL : ('\r')? '\n';
+fragment QUOTED_DQ : DQ DQ;
+fragment QUOTED_SQ : SQ SQ;
+fragment DQ : '"';
+fragment SQ : '\'';
+fragment INCLUDE : I N C L U D E;
+
+fragment A : [aA];
```

Comment:
Most of fragment are not used. Lexer rule can be defined in an easier way.

Reply:
File is tiny anyway

Reply:
If you want to keep unused fragments for letters at least centralize them in a single and you can used these lexer rules in all your files. 

```
lexer grammar MyLexer;
```

```
options { tokenVocab= MyLexer; }
```